recipes/sox/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX
+make
+make bindir=. installcheck
```

Comment:
Sorry. Yes that's what I meant. There is no `make check` but there is `make
bindir=. installcheck` which does the same thing.

On Thu, Mar 31, 2016 at 1:09 PM Filipe <notifications@github.com> wrote:

> In recipes/sox/build.sh
> <https://github.com/conda-forge/staged-recipes/pull/239#discussion_r58036507>
> :
>
> > @@ -0,0 +1,6 @@
> > +#!/bin/bash
> > +
> > +./configure --prefix=$PREFIX --exec-prefix=$PREFIX
> > +make
> > +make bindir=. installcheck
>
> That means no make check? I am not familiar with that syntax. (Due to the
> high traffic of messages in conda-forge I'd appreciate if you can be more
> verbose rather than copying and pasting a line.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/239/files/b77c168fc849ebd8551ca0ba335e06cce961deb1#r58036507>
>